ocpp/v16/ocpp_factory.py

```diff
@@ -77,10 +77,10 @@ def remove_nones(dict_to_scan):
     return dict_to_scan
 
 
-class ChargePoint:
+class OcppFactory:
```

Comment:
I found this name confusing. Why did you choose this name? The literal translation can be confusing

Reply:
In computer science `factory` has a specific meaning. It's often used for a structure that allows to build certain components. 
See https://en.wikipedia.org/wiki/Factory_method_pattern

Reply:
thanks, @OrangeTux ! I did not know that.